css/shared/colors.css

```diff
@@ -587,6 +595,14 @@
 
 /* background colors */
 
+.bg-white {
```

Comment:
here also I got confused. 
If you are creating color variables then you could simple do  --pink-100 : color name or code instead of putting it in a class.